Kudu.Services.Web/Content/Scripts/FileBrowser.js

```diff
@@ -368,8 +360,7 @@ $.connection.hub.start().done(function () {
             viewModel.editText('');
             statusbar.fetchingContents();
             viewModel.fileEdit(this);
-            if(this.mime === "text/xml")
-            {
+            if (this.mime === "text/xml") {
                 Vfs.getContent(this)
                    .done(function (data) {
                        viewModel.editText(vkbeautify.xml(data));
```

Comment:
Nope, i was wrong. Setting XML mode in Ace does not beautify the content, only enables coloring. Let's just leave that in.